django/cantusdb_project/main_app/templates/melody_search.html

```diff
@@ -13,7 +13,7 @@ <h3>Search by melody</h3>
                 go here: <a href="https://cantusindex.org/melody" target="_blank">Cantus Index Melody Search</a>.
             </p>
 
-            <div id="drawArea" style="font-size:0">
+            <div id="drawArea" class="text-truncate" style="font-size:0">
```

Comment:
I can't find `text-truncate` anywhere else in the codebase. Is this a thing that bootstrap provides?

Reply:
Yes, this is bootstrap. To be honest, I don't quite understand why it worked, and I only saw this (https://getbootstrap.com/docs/5.0/helpers/text-truncation/) but didn't look at the underlying css. 

I think I will add a comment here...we can put it up for time's sake but we may want to clean it up later?